assets/canary/role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: canary-role
+  namespace: openshift-ingress-canary
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: canary-role
+subjects:
+- kind: ServiceAccount
+  name: ingress-operator
```

Comment:
Does the canary daemonset need to be configured to use this service account?

Reply:
At the moment, no. The canary-role gives the ingress-operator service account (and thus, the operator) extended permissions for routes in the `openshift-ingress-canary` namespace only. The operand (hello-openshift in this case) does not need any RBAC permissions.